lib/vector-buffers/src/buffer_usage_data.rs

```diff
@@ -95,16 +100,25 @@ pub struct BufferUsageData {
     received_byte_size: AtomicU64,
     sent_event_count: AtomicU64,
     sent_byte_size: AtomicU64,
-    dropped_event_count: Option<AtomicU64>,
+    dropped_event_data: Option<BufferUsageDroppedEventData>,
     max_size_bytes: AtomicU64,
     max_size_events: AtomicUsize,
 }
 
+#[derive(Debug)]
+struct BufferUsageDroppedEventData {
+    dropped_event_count: AtomicU64,
+    dropped_event_size: AtomicU64,
```

Comment:
Does this need the `dropped_event_` prefix, since that's part of the structure name?

Reply:
Yeah, that's a good point 👍